superset/assets/stylesheets/less/variables.less

```diff
@@ -107,9 +107,21 @@
 /* Z-INDEX                                                              */
 /* Think of the site as "layers" rather than an arms race of numbers    */
 /* Try to keep these to a minimum                                       */
+/* Try to label semantic "layers" and add comments for usage notes      */
+/* Use double dash modifiers to step up/down from a base layer          */
+/* e.g. z-whatever--modifier                                            */
 /************************************************************************/
-@z-index-max: 1000;
-@z-index-dropdown: 11;
+
+/************************ toast messages, popovers **********************/
+@z-max: 3000;
```

Comment:
any reason you got rid of `index` in these var names? I kinda liked it in there

Reply:
Just felt so dash-tastic with them in there! I figured just `z` captured the intent enough.

That said, I don't have a strong opinion. `z-index-...` was my first notion, so I'm glad to hear you liked it... I'll put it back!